src/components/ui/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,57 @@
+import { Input as UIInput, Label as UILabel } from '@faststore/ui'
+import { forwardRef } from 'react'
+import type { InputHTMLAttributes } from 'react'
+import Icon from 'src/components/ui/Icon'
+
+export interface ToggleProps
+  extends InputHTMLAttributes<Omit<HTMLInputElement, 'disabled' | 'type'>> {
+  /**
+   * ID to identify input and corresponding label.
+   */
+  id: string
+  /**
+   * The text displayed to identify the input.
+   */
+  label?: string
```

Comment:
I don't think this should optional. It's required that the input is labeled, either by a connected `<label>` (visible) or `aria-label` (visible only to screen readers).

Currently, when no `label` is given, the `input` is being labeled via [`aria-label` with `toggle-${id}`](https://github.com/vtex-sites/gatsby.store/blob/f28d03b3ad7b1886c53317e08ac02b0bfed59b53/src/components/ui/Toggle/Toggle.tsx#L35), but that's not a good label for screen reader users. The label should humanly indicate what exactly is being toggled.

Not sure what's the best solution here yet, e.g. keep `label` required but have another prop to indicate whether to show the label (`<label>`) or not (`aria-label`)?

Reply:
WDYT about a `aria-label` value more descriptive like "Toggle input with ID <id>"? Talked with @renatamottam and she also prefers that the `label` prop shouldn't be required.

Reply:
Not sure I like it as "ID" is an identifier for computers, not humans... Let's see if @hellofanny has any suggestions!

Reply:
Hmm, I agree with what Filipe said. Can't we keep the `label`? The person might not know what ID are the input referring to.